foundation/aissemble-foundation-core-python/pyproject.toml

```diff
@@ -21,6 +21,8 @@ krausening = ">=20"
 cryptography = ">=42.0.4"
 pyjwt = ">=2.3.0"
 pyjks = ">=20.0.0"
+pyasn1 = "0.5.1"
```

Comment:
S: We should provide a range that caps at 0.5.1/0.4.0 and provide a comment explaining why we're capping it.

Reply:
Based on existing lock file probably `>=0.3.5,<0.6.0` and `>=0.0.8,<0.4.1'